pkg/values/validation/schemas.go

```diff
@@ -0,0 +1,306 @@
+//            - Deleted
+//      properties:
+//        name:
+//          type: string
+//        apiVersion:
+//          type: string
+//        kind:
+//          type: string
+//        includeSnapshotsFrom:
+//          type: array
+//          additionalItems: false
+//          minItems: 1
+//          items:
+//            type: string
+//        queue:
+//          type: string
+//        jqFilter:
+//          type: string
+//          example: ".metadata.labels"
+//        keepFullObjectsInMemory:
+//          type: boolean
+//        allowFailure:
+//          type: boolean
+//        executeHookOnSynchronization:
+//          type: boolean
+//        waitForSynchronization:
+//          type: boolean
+//        resynchronizationPeriod:
+//          type: string
+//        nameSelector:
+//          "$ref": "#/definitions/nameSelector"
+//        labelSelector:
+//          "$ref": "#/definitions/labelSelector"
+//        fieldSelector:
+//          type: object
+//          additionalProperties: false
+//          required:
+//          - matchExpressions
+//          properties:
+//            matchExpressions:
+//              type: array
+//              items:
+//                type: object
+//                additionalProperties: false
+//                minProperties: 3
+//                maxProperties: 3
+//                properties:
+//                  field:
+//                    type: string
+//                  operator:
+//                    type: string
+//                    enum: ["=", "==", "Equals", "!=", "NotEquals"]
+//                  value:
+//                    type: string
+//        group:
+//          type: string
+//        namespace:
+//          type: object
+//          additionalProperties: false
+//          minProperties: 1
+//          maxProperties: 2
+//          properties:
+//            nameSelector:
+//              "$ref": "#/definitions/nameSelector"
+//            labelSelector:
+//              "$ref": "#/definitions/labelSelector"
+//`,
+//	"v0": `
+//type: object
+//additionalProperties: false
+//minProperties: 1
+//properties:
+//  onStartup:
+//    title: onStartup binding
+//    description: |
+//      the value is the order to sort onStartup hooks
+//    type: integer
+//  schedule:
+//    type: array
+//    items:
+//      type: object
+//  onKubernetesEvent:
+//    type: array
+//    items:
+//      type: object
+//`,
+//}
+
+type ValuesSchemaType string
+
+const (
+	GlobalSchema       ValuesSchemaType = "global"
+	ModuleSchema       ValuesSchemaType = "module"
+	ConfigValuesSchema ValuesSchemaType = "config"
+	MemoryValuesSchema ValuesSchemaType = "memory"
+)
+
+var GlobalSchemasCache = map[string]*spec.Schema{}
+var ModuleSchemasCache = map[string]map[string]*spec.Schema{}
+
+// GetGlobalValuesSchema returns ready-to-use schema for global values.
+func GetGlobalValuesSchema(valuesType string) *spec.Schema {
+	if s, ok := GlobalSchemasCache[valuesType]; ok {
```

Comment:
It can be simplyfied by `return GlobalSchemasCache[valuesType]`.

Reply:
Done